File: src/main/java/ar/edu/unq/epers/unidad5/model/Precio.java
package ar.edu.unq.epers.unidad5.model;

import lombok.*;

@ToString
@Setter
@Getter
@EqualsAndHashCode
@NoArgsConstructor(access = AccessLevel.PROTECTED)

public class Precio {

    private Zona zona;
    private Usuario user;
    private int precio;

    public Precio(Zona zona, Usuario user, int precio) {
        this.zona = zona;
        this.user = user;
        this.precio = precio;
    }
}

File: src/main/kotlin/ar/edu/unq/epers/unidad5/model/Precio.kt
package ar.edu.unq.epers.unidad5.model

class Precio {
    var zona: Zona? = null
    var user: Usuario? = null
    var precio = 0


    //Se necesita un constructor vacio para que jackson pueda
    //convertir de JSON a este objeto.
    protected constructor() {}
    constructor(zona: Zona?, user: Usuario?, precio: Int, producto: Producto?) {
        this.zona = zona
        this.user = user
        this.precio = precio
    }
}